### PR TITLE
Include partial causatives in general report

### LINKED ISSUE #1587: Include partial causative variants in general report
They're not shown at the moment

### PR DESCRIPTION
**Pull request by northwestwitch**
fix #1587

**How to test**:
1. Mark one or more variants (SNVs, SVs) as partial causatives
1. Go to general report and check they're shown on it (also relative HPO and OMIM terms)
1. Make sure that also the PDF report is correct.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by DN, CR

### CODE REVIEW

File: scout/server/blueprints/cases/controllers.py [hunk 1]
```diff
@@ -263,16 +264,17 @@ def case_report_content(store, institute_obj, case_obj):
     data['report_created_at'] = datetime.datetime.now().strftime("%Y-%m-%d %H:%M")
 
     evaluated_variants = {vt:[] for vt in variant_types}
-    # We collect all causatives and suspected variants
+    # We collect all causatives (including the partial ones) and suspected variants
     # These are handeled in separate since they are on case level
-    for var_type in ['causatives', 'suspects']:
+    for var_type in ['causatives', 'suspects', 'partial_causatives']:
```
Comment (dnil) [inline scout/server/blueprints/cases/controllers.py]: :+1:

File: scout/server/blueprints/cases/templates/cases/case_report.html [hunk 2]
```diff
@@ -857,240 +868,24 @@ <h4>Scout case analysis report</h4> - created on:&nbsp;<strong>{{report_created_
 </div>
 {% endmacro %}
 
-{% macro sv_variant_content(variant,index) %} <!--The entire description of a structural variant, dynamic contentc -->
```
Comment (dnil) [inline scout/server/blueprints/cases/templates/cases/case_report.html]: Eh, what happened there?!

Comment (northwestwitch) [reply scout/server/blueprints/cases/templates/cases/case_report.html]: If you open master you'll see that there are 2 sv_variant_content macros on the same page. Of course only the last one was used in the end. I must have forgotten one when I touched this code last!

Comment (dnil) [reply scout/server/blueprints/cases/templates/cases/case_report.html]: Ok! Looks good when looking at the whole file, but really weird on the diff! :blush:

**Comment by northwestwitch** [comment]
Tested on stage and it works, both the HTML code and the PDF:

image

**Comment by northwestwitch** [comment]
One thing that is actually not rendering well on the PDF version is the Font Awesome icons (above is the HTML page, below the PDF):

image

I'm going to convert them to text instead!

**Comment by northwestwitch** [comment]
Now it's better:

image

**Pull request merged**